src/server/scripts/DragonIsles/AberrusTheShadowedCrucible/boss_kazzara_the_hellforged.cpp

```diff
@@ -137,7 +95,9 @@ struct boss_kazzara_the_hellforged : public BossAI
 
                 scheduler.Schedule(10s, [this](TaskContext /*context*/)
                 {
-                    me->RemoveUnitFlag(UnitFlags(UNIT_FLAG_IMMUNE_TO_PC | UNIT_FLAG_IMMUNE_TO_NPC | UNIT_FLAG_UNINTERACTIBLE));
+                    me->RemoveUnitFlag(UnitFlags(UNIT_FLAG_UNINTERACTIBLE));
```

Comment:
@Shauren should we not create a setter for SetUninteractible(bool) as well?

Reply:
Merged, it can be used now!

Reply:
done